guides/whats-new.md

```diff
@@ -0,0 +1,214 @@
+
+`$w.slider` 
+
+The [`value`](https://www.wix.com/corvid/reference/$w/slider/value) property for the Slider element was described as read only but it should be read and write.
+
+## 🗓️ September 15, 2020
+### Updated: Added more information for returned object properties from Validity property
+
+`ValidatableMixin`
+
+Added more content to the table that displays details of the object returned by [`validity`](https://www.wix.com/corvid/reference/$w/validatablemixin/validity).
+
+## 🗓️ September 14, 2020
+### Fixed: Added information that some getProductVariants parameter fields are optional
+
+`wix-stores`
+
+Added information that some of the parameter fields for [getProductVariants](https://www.wix.com/corvid/reference/wix-stores/getproductvariants) are optional.
+
+## 🗓️ September 14, 2020
+### Fixed: createProduct SKU is a string
+
+`stores-backend`
+
+The `sku` property for [createProduct](https://www.wix.com/corvid/reference/wix-stores-backend/createproduct) was described as a number, but it is a string.
+
+## 🗓️ September 10, 2020
+### Updated: Added information about getFileURL token expiring
+
+`wix-media-backend`
+
+Added information that the token returned by [`getFileUrl()`](https://www.wix.com/corvid/reference/wix-media-backend/mediamanager-obj/getfileurl) expires after 600 minutes. Described the alternative of creating a static URL without a token.
+
+## 🗓️ September 10, 2020
+### Fixed: The URL table for wixHttpFunctionRequest
+
+`wix-http-functions`
+
+The table that described the URL structure of a [wixHttpFunctionRequest](https://www.wix.com/corvid/reference/wix-http-functions/wixhttpfunctionrequest) was difficult to parse. We've updated the content to make it easier to understand. 
+
+## 🗓️ September 7, 2020
+### New API: On cart changed event
+
+`wix-stores`
+
+With the new [`onCartChanged()`](https://www.wix.com/corvid/reference/wix-stores/oncartchanged) client-side event in wix-stores, you can access information about the site's shopping cart every time an item is added or removed. Useful for creating a promotional lightbox - if product X is added to the cart, offer product Y.
+
+## 🗓️ September 7, 2020
+### New API: Consent policy (privacy laws)
+
+`wix-users`
+
+To help your site conform to GDPR and CCPA standards, we now provide Corvid APIs for [checking](https://www.wix.com/corvid/reference/wix-users/getcurrentconsentpolicy) and [setting](https://www.wix.com/corvid/reference/wix-users/setconsentpolicy) visitors' current consent policies. These policies include which cookies the visitor allows and if the visitor allows data transfer to third parties. 
+
+## 🗓️ August 30, 2020
+### Updated: Added examples to listMemberBadges
+
+`wix-users-backend`
+
+Added more robust examples to [`listMemberBadges()`](https://www.wix.com/corvid/reference/wix-users-backend/badges-obj/listmemberbadges) for how to handle the results.
+
+
+## 🗓️ August 23, 2020
+### New API: Coupons limit per customer
```

Comment:
I know the API is plural (coupons) but would keep it single here for readability